docs/contributions/rfc_process.md

```diff
@@ -0,0 +1,113 @@
+*   Features that create new library dependencies for Cirq.
+*   New simulators, samplers, protocols, devices, or packages.
+*   Features that drastically change performance of Cirq.
+*   Features that maintainers disagree/debate about.
+
+Not Major features include the following:
+
+
+
+*   Fixing a bug.
+*   Extending the functionality of an existing method in a natural way.
+
+If you are not sure if a feature constitute as a “major feature”, just submit a github issue with a description, and one of the maintainers will flag the issue as a major feature if necessary
+
+
+## How to submit an RFC
+
+
+
+1. Before submitting an RFC, discuss your aims with project contributors and maintainers and get early feedback.  Maintainers can be contacted at cirq-maintainers@googlegroups.com
+2. Draft your RFC.
+    *   Follow the [RFC template](https://docs.google.com/document/d/1Zf0n7rKahejSJmAgYbCP6ClKXe8HhALceRQb-vRYe6o/edit#heading=h.mckr847qq60k).
+3. After writing the RFC draft, get feedback from maintainers and contributors before submitting it (email: cirq-maintainers@googlegroups.com) \
+Writing implementation code is not a requirement, but it may help design discussions.
+4. Recruit a sponsor.
+    *   A sponsor must be a maintainer of the project or the product manager
+    *   Identify the sponsor in the RFC, before submitting a Github issue.
+5. Attend the Cirq Cync to review your RFC (Cirq Cyncs happen once per week, Google Calendar [event](https://calendar.google.com/event?action=TEMPLATE&tmeid=NzBrdjdxbGFraTN2ajUyanJoaWRuZXJuMWcgY180am1mZmdjNDVsYXVucXU2a3JiM3JrNDhzNEBn&tmsrc=c_4jmffgc45launqu6krb3rk48s4%40group.calendar.google.com))
+6. The meeting may approve the RFC, reject it, or require changes before it can be considered again. Approved RFCs will be updated in Github by the sponsor.
+
+
+## RFC participants
+
+Many people are involved in the RFC process:
+
+
+
+*   **RFC author** — one or more community members who write an RFC and are committed to championing it through the process
+*   **RFC sponsor** — a maintainer or product manager who sponsors the RFC and will shepherd it through the RFC review process
+*   **review committee** — a group of maintainers who have the responsibility of recommending the adoption of the RFC.  This group can be emailed by emailing cirq-maintainers@googlegroups.com. This is typically the attendees at the Cirq Cync meeting.
+*   Any **community member** may help by providing feedback on whether the RFC will meet their needs.
+
+
+### RFC sponsors
+
+A sponsor is a project maintainer responsible for ensuring the best possible outcome of the RFC process. This includes:
+
+
+
+*   Advocating for the proposed design.
+*   Guiding the review committee to come to a productive consensus.
+*   If the RFC moves to implementation:
+    *   Ensuring proposed implementation adheres to the design.
+    *   Coordinate with appropriate parties to successfully land implementation.
```

Comment:
I feel like we should also capture prioritization here and that, if the feature is large enough, it can be represented on the Roadmap as well